framework/components/ecl-navigation/_navigation.scss

```diff
@@ -0,0 +1,64 @@
+/**
+ * ECL Navigation
+ * @define navigation
+ */
+
+$ecl-navigation-border-width: 2px !default;
+
+.ecl-navigation {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  list-style-type: none;
+  padding: 0;
+}
+
+.ecl-navigation--tabs {
+  border-bottom: $ecl-navigation-border-width solid map-get($ecl-colors, 'grey-50');
+}
+
+.ecl-navigation__item {
+  flex-shrink: 0;
+
+  .ecl-navigation--tabs & {
+    margin-bottom: -$ecl-navigation-border-width;
+  }
+}
+
+.ecl-navigation__item-link {
```

Comment:
we have to synchronize here: I named it __link in the horizontal menu, and not __item-link.
So which one do we keep?
I would go for __link as it is descriptive enough